### DISCUSSION TITLE
Headreco command not found in matlab (brainstorm)

### DISCUSSION THREAD

**Discussion by nikiita013** [simnibs/simnibs#342] [2. General]
Hi,
I installed simnibs 3.2.6 and the headreco command works on the terminal. However, when I run it via brainstorm on matlab then it gives following error:
```
** Error: SimNIBS 3.x is not installed or not added to the system path:
** the command "headreco" could not be found.
```

I tried to add to path` * addpath('/home/user/SimNIBS-3.2/simnibs/matlab/')* `

and also ran this` 'setenv('LD_LIBRARY_PATH', sprintf('/home/user/SimNIBS-3.2/simnibs/bin/linux:%s', getenv('LD_LIBRARY_PATH')))'`

Still it is giving me the same error. When I simply write on terminal headreco I get following:
```
usage: headreco [-h] [--version] mode ...
headreco: error: the following arguments are required: mode
```

Any help would be greatly appreciated

**Comment by oulap** [comment]
The command line interfaces (like headreco) should be located in 

/home/user/SimNIBS-3.2/bin

or some similar location based on where you installed simnibs. Note that it's not this location:

/home/user/SimNIBS-3.2/simnibs/bin/linux

that's where the linux specific dependencies are. Try adding the first path to your system path in matlab and see if that works.

**Reply by nikiita013** [reply]
Thanks a lot! It worked 👍   'setenv('PATH', sprintf('/home/user/SimNIBS-3.2/bin:%s', getenv('PATH')))'
